MapboxSpeechTests/MapboxVoiceTests.swift

```diff
@@ -38,6 +38,7 @@ class MapboxVoiceTests: XCTestCase {
         let options = SpeechOptions(text: "hello")
         options.outputFormat = .mp3
         options.speechGender = .female
+        options.locale = Locale(identifier: "en_US")
```

Comment:
My simulator was set to en-gb, so tests failed. This makes sure this does not happen in the future.